src/components/DeckBuilderDryRunView/index.js

```diff
@@ -10,10 +12,14 @@ export default props => {
   // purposes. The mode is restored to `AUTOMATIC` as soon as the 4th card has
   // been picked.
   const [mode, setMode] = React.useState(params.get('mode') || 'AUTOMATIC')
+  const [modifier, setModifier] = React.useState('0')
   const [equalsMode, setEqualsMode] = React.useState(false)
-  const deck = equalsMode
-    ? props.deck.map(card => ({ ...card, level: 1 }))
-    : props.deck
+  const fullDeck = props.deck.map(card =>
+    equalsMode
+      ? resolveCardForLevel({ ...card, level: 1 })
+      : resolveCardForLevel(card)
+  )
+  const deck = modifyDeck(fullDeck, modifier)
```

Comment:
I feel like the `equalsMode` modification could happen in that function. That’s a good idea to have extracted it by the way.